README.md

```diff
@@ -17,13 +17,16 @@
 
 ## Building
 
-Go 1.6 is required to build confd, which uses the new vendor directory.
+To Build confd and run the BIRD template tests:
 
+```shell
+$ make all
 ```
-$ mkdir -p $GOPATH/src/github.com/kelseyhightower
-$ git clone https://github.com/kelseyhightower/confd.git $GOPATH/src/github.com/kelseyhightower/confd
-$ cd $GOPATH/src/github.com/kelseyhightower/confd
-$ ./build
+
+Or to simply build confd:
+
+```shell
+$ make vendor && make bin/confd
```

Comment:
It's probably worth updating the stuff at the top of the readme file too - since it references the original repo (including the badge).

Probably should just remove all of that and state that this is a fork of the blah project to add kubernetes support specific to Calico.